src/openeo_test_suite/lib/compliance_util.py

```diff
@@ -35,6 +35,8 @@ def test_endpoint(
     method: str = "GET",
     expected_status_codes: Union[list[int], int] = [200],
     return_response: bool = False,
+    mock_response_content: bytes = None,
```

Comment:
I don't seem to find where this `mock_response_content` is being used, is it necessary to add this?